docs/source/conf.py

```diff
@@ -0,0 +1,56 @@
+# Configuration file for the Sphinx documentation builder.
+#
+# This file only contains a selection of the most common options. For a full
+# list see the documentation:
+# https://www.sphinx-doc.org/en/master/usage/configuration.html
+
+# -- Path setup --------------------------------------------------------------
+
+# If extensions (or modules to document with autodoc) are in another directory,
+# add these directories to sys.path here. If the directory is relative to the
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+#
+# import os
+# import sys
+# sys.path.insert(0, os.path.abspath('.'))
+
+
+# -- Project information -----------------------------------------------------
+
+project = "pyhf-benchmark"
+copyright = "2020, Bo Zheng"
+author = "Bo Zheng"
+
+# The full version, including alpha/beta/rc tags
+release = "0.0.1"
+
+
+# -- General configuration ---------------------------------------------------
+
+# Add any Sphinx extension module names here, as strings. They can be
+# extensions coming with Sphinx (named 'sphinx.ext.*') or your custom
+# ones.
+extensions = ["m2r2"]
```

Comment:
This is fine for now, but given that [`m2r2`](https://github.com/CrossNox/m2r2#m2r-the-original) is a fork of an abandoned project it is probably worth just switching to RST for long term support, which `pyhf` reluctantly did. That can be for another PR though. :+1:

Reply:
Okay, since I wanted to use README.md directly rather than write README.rst before. I will discard the usage of m2r2 and wrote a README.rst file in the next PR.

Reply:
Yeah keeping it as Markdown I think is fine for now.